src/javascript/containers/AgreementList.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router'
+import { sendMoney, confirmTransaction, cancelTransaction } from '../actions/agreementList';
+import Agreement from '../components/agreement';
+import type { AgreementsType } from '../types';
+
+class AgreementList extends Component {
+  // map over all agreements using allTransactions as values
+  // preferably, use some sort of sorting function to return
+  // date, amount, as buyer, as seller, completed
+
+  render() {
+      return (
+        <ul>
+          <Agreement {...this.props} />
+        </ul>
+      );
+    }
+  };
+
+  const mapStateToProps = (state) => {
```

Comment:
Should this lambda have types?

`const mapStateToProps = (state: ???): ??? => {`